distribution/contents/conf/wso2/wso2as-web.xml

```diff
@@ -12,10 +12,10 @@
  limitations under the License.
 -->
 <wso2as-web xmlns="http://wso2.org/2016/wso2as-web">
-    <classloading>
-        <parent-first>true</parent-first>
+    <class-loader>
+        <parent-first>false</parent-first>
         <environments>CXF</environments>
```

Comment:
We don't need to add tomcat it self as a environment. I will remove this in next PR